includes/adapter/delivery-options-from-order-adapter.php

```diff
@@ -15,10 +15,10 @@ class WCMP_DeliveryOptionsFromOrderAdapter extends AbstractDeliveryOptionsAdapte
      */
     public function __construct(?AbstractDeliveryOptionsAdapter $originAdapter, array $inputData = [])
     {
-        $adapterCarrier        = $originAdapter ? $originAdapter->getCarrier() : null;
+        $adapterCarrier        = $originAdapter ? $originAdapter->getCarrier() : WCMP_Data::DEFAULT_CARRIER_NAME;
         $adapterDate           = $originAdapter ? $originAdapter->getDate() : null;
         $adapterDeliveryType   = $originAdapter ? $originAdapter->getDeliveryType() : AbstractConsignment::DEFAULT_DELIVERY_TYPE_NAME;
-        $adapterPackageType    = $originAdapter ? $originAdapter->getPackageType() : null;
+        $adapterPackageType    = $originAdapter ? $originAdapter->getPackageType() : AbstractConsignment::DEFAULT_PACKAGE_TYPE_NAME;
```

Comment:
Kan niet wachten tot ~2030~ 2023 🤩, wanneer heel php 7 EOL is en we dit mogen doen:

```php
return $originAdapter?->getPackageType();
``` 
_(Waar het niet dat WordPress in 2030 waarschijnlijk nog steeds op php 7 zit...)_